modules/eventschedule.php

```diff
@@ -178,8 +178,8 @@ function parseWorkingHours($hours_period)
 $userlistcount = count($userid);
 
 $params['short'] = 1;
-if (ConfigHelper::getConfig('phpui.timetable_hide_disabled_users')) {
-    $params['access'] = 1;
+if (ConfigHelper::checkConfig('phpui.timetable_hide_disabled_users')) {
+    $params['userAccess'] = 1;
 }
```

Comment:
Czy to czasem nie powinno pójść oddzielnym commitem i PR-em, żeby móc przenieść do wersji stabilnych?
Do jakich ewentualnie wersji stabilnych?

Reply:
ta zmiana będzie miała wpływ na metodę $LMS->GetUserList(); i tak w ogóle to spore zmiany jednocześnie do terminarza i planu, więc ja bym tego nie pchał do stable, bo to co tam jest działa, a ta zmiana to 'improvement'. Tym bardziej, że teraz wydania stabilne są częstsze.

Reply:
W jaki sposób ta jedna zmiana związana tylko z phpui.timetable_hide_disabled_users wpływa na resztę?
Moim zdaniem, gdyby tylko tę zmianę powyżej przenieść do stable to mogłoby być to uznane za pewną poprawkę, bo getConfig() na wartości logicznej to trochę prowizorka.. Nie piszę o całym PR, lecz tylko o tej pojedynczej zmianie.
No i jeśli miałoby to być przeniesione do **stable** to przydałby się na to oddzielny PR.

Reply:
> W jaki sposób ta jedna zmiana związana tylko z phpui.timetable_hide_disabled_users wpływa na resztę?

Wpływa tylko na zachowanie metody $LMS->GetUserList() 

> getConfig() na wartości logicznej to trochę prowizorka

Zgadza się. Dopiero niedawno to odkryłem :) Przygotuję oddzielny PR.